File: android/src/main/kotlin/com/kardenvan/sber_auth_sdk_flutter/login/parameters/SberIdLoginParameters.kt
package com.kardenvan.sber_auth_sdk_flutter.login.parameters

data class SberIdLoginParameters constructor(
    val clientId: String,
    val scope: String,
    val state: String,
    val nonce: String,
    val returnUrl: String,
    val codeChallenge: SberIdLoginCodeChallenge?,
    val customTabsRedirectUrl: String?,
)

data class SberIdLoginCodeChallenge constructor(
    val value: String,
    val method: String,
)

File: ios/Classes/login/parameters/SberIdLoginParameters.swift
//
//  SberIdLoginParameters.swift
//  sber_auth_sdk_flutter
//
//  Created by apple on 16.03.2023.
//

import Foundation

struct SberIdLoginParameters {
    let clientId: String
    let scope: String
    let state: String
    let nonce: String
    let returnUrl: String
    let codeChallenge: SberIdLoginCodeChallenge?
    let webPageAuthUrl: String?
}

struct SberIdLoginCodeChallenge {
    let value: String
    let method: String
}
